src/main/java/dk/bec/polonez/reservationsystem/service/MailService.java

```diff
@@ -0,0 +1,60 @@
+package dk.bec.polonez.reservationsystem.service;
+
+import dk.bec.polonez.reservationsystem.dto.notificationDto.ReservationConfirmationInput;
+import dk.bec.polonez.reservationsystem.dto.notificationDto.ReservationStatusNotificationInput;
+import dk.bec.polonez.reservationsystem.dto.notificationDto.UpcomingEventsNotificationInput;
+import dk.bec.polonez.reservationsystem.model.NotificationTemplate;
+import dk.bec.polonez.reservationsystem.repository.NotificationTemplateRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class MailService {
+
+    private final JavaMailSender mailSender;
+
+    private final NotificationTemplateRepository templateRepository;
+
+    public void sendReservationStatusNotification(String emailAddress, ReservationStatusNotificationInput input) {
```

Comment:
a co z testami?

juz ktorys raz o to pytam

Reply:
@Achreko i @AleXand3rG piszą testy od wczoraj, wleci cały pull request pełen testów do wszystkiego.

Reply:
ok

ogolnie to w kazdym PR powinny byc testy do tego co dodajecie, bo tak to nawet nie ma pewnosci, czy to co jest napisane dziala ;)

Reply:
zgadzam się, ale mieliśmy problem z podziałem zadań w projekcie i gdyby każdy pisał testy do swoich featurów to dwie osoby nie miałby co robić. Testujemy na bieżąco działanie endpointów w Postmanie.